### ISSUE TITLE
New Honeywell integration only shows one of two thermostats in My Account

### ISSUE THREAD

**Issue by Trebacz** [home-assistant/home-assistant.github.io#18759]
### Feedback

I upgraded to the new integration. One of my thermostats cam over perfect. I don't see the second one.

![image](https://user-images.githubusercontent.com/19363347/128278790-d9b73901-4950-4e60-ae7e-e45091c8f929.png)

### URL

https://www.home-assistant.io/integrations/honeywell/

### Version

2021.8.0

**Comment by Trebacz** [comment]
Actually the second thermostat magically appeared after a restart of HA - followed by waiting (maybe about 20 minutes? Now it's there like it was. The integration on shows one device.
![image](https://user-images.githubusercontent.com/19363347/128281507-cdd9b628-b229-4e80-9aa5-518e8c2f9a36.png)
![image](https://user-images.githubusercontent.com/19363347/128281581-8214c8ec-c42b-4d7a-b947-296154848711.png)
Not sure what's up?

**Comment by frenck** [comment]
Hi there @Trebacz 👋 

Two things.

It seems like you want to report an issue with the integration, however, you have submitted documentation feedback instead. If you want to report an issue, please use this issue tracker instead:
<https://github.com/home-assistant/core/issues>

This is a polling integration, it figures out / detects the thermostats on the first poll after startup. So, that sounds like exactly what it did?

Anyways, if you still think it is an issue, please use the first link to report it.

Thanks 👍 

../Frenck

**Comment by rdfurman** [comment]
Dupe of https://github.com/home-assistant/core/issues/53981.

**Comment by ejbernard64** [comment]
@frenck.  First, thanks for all that you do for HA that enables us novice users to really enjoy what the product provides.  I've learned a lot from you on this site as well as the monthly release video updates.

With respect to you comment to @Trebacz above, the polling makes sense as a possible cause.  I have 9 thermostats in my house.  It appears that only the first came in after upgrading to 2021.08.4.  I also suspected this may be a polling issue so I decided to just let things catch up.  Well, its been more than 24 hours now and no change.  Still have only 1 of 9 thermostats.  My family uses HA to manage their own room thermostat.  I may be forced to roll back.  Is it possible to force a poll update?  Thanks in advance.
